edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.adoc

```diff
@@ -0,0 +1,130 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="understanding-image-based-upgrade-for-sno"]
+= Understanding the image-based upgrade for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: understanding-image-based-upgrade
+
+toc::[]
+
+:FeatureName: The Lifecycle Agent
+
+From {product-title} 4.14.13, the {lcao} provides you with an alternative way to upgrade the platform version of a {sno} cluster.
+The image-based upgrade is faster than the standard upgrade method and allows you to directly upgrade from {product-title} <4.y> to <4.y+2>, and <4.y.z> to <4.y.z+n>.
+
+// Lifecycle Agent (LCAO)
+
+This upgrade method utilizes a generated OCI image from a dedicated seed cluster that is installed on the target {sno} cluster as a new `ostree` stateroot.
+A seed cluster is a {sno} cluster deployed with the target {product-title} version, Day 2 Operators, and configurations that are common to all target clusters.
+
+You can use the seed image, which is generated from the seed cluster, to upgrade the platform version on any {sno} cluster that has the same combination of hardware, Day 2 Operators, and cluster configuration as the seed cluster.
+
+[IMPORTANT]
+====
+The image-based upgrade uses custom images that are specific to the hardware platform that the clusters are running on.
+Each different hardware platform requires a separate seed image.
+====
+
+The {lcao} uses two custom resources (CRs) on the participating clusters to orchestrate the upgrade:
+
+* On the seed cluster, the `SeedGenerator` CR allows for the seed image generation. This CR specifies the repository to push the seed image to.
+* On the target cluster, the `ImageBasedUpgrade` CR specifies the seed image for the upgrade of the target cluster and the backup configurations for your workloads.
+
+.Example SeedGenerator CR
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1
+kind: SeedGenerator
+metadata:
+  name: seedimage
+spec:
+  seedImage: <seed_image>
+----
+
+.Example ImageBasedUpgrade CR
+[source,yaml]
+----
+apiVersion: lca.openshift.io/v1
+kind: ImageBasedUpgrade
+metadata:
+  name: upgrade
+spec:
+  stage: Idle <1>
+  seedImageRef: <2>
+    version: <target_version>
+    image: <seed_container_image>
+    pullSecretRef:
+      name: <seed_pull_secret>
+  autoRollbackOnFailure: {}
+#    initMonitorTimeoutSeconds: 1800 <3>
+  extraManifests: <4>
+  - name: example-extra-manifests
+    namespace: openshift-lifecycle-agent
+  oadpContent: <5>
+  - name: oadp-cm-example
```

Comment:
We are providing an example name throughout the procedure. It can be changed if needed but it's easier to copypaste if someone just wants to try the feature, so it's for convenience. "example" is in the name to make it obvious that it's not required to use the same name.